src/lib/browse/layers/PctLayerControl.svelte

```diff
@@ -0,0 +1,130 @@
+<script lang="ts">
+  import { ExternalLink, HelpButton, InteractiveLayer } from "lib/common";
+  import { Checkbox, Radio, Select } from "lib/govuk";
+  import {
+    hoveredToggle,
+    makeColorRamp,
+    overwriteLineLayer,
+    overwritePmtilesSource,
+  } from "lib/maplibre";
+  import type { MapGeoJSONFeature } from "maplibre-gl";
+  import { map } from "stores";
+  import { colors, denseLineWidth } from "../colors";
+  import SequentialLegend from "./SequentialLegend.svelte";
+
+  // TODO It'd be much simpler to have one source with both attributes
+  let nameCommute = "pct_commute";
+  let nameSchool = "pct_school";
+  let colorScale = colors.sequential_low_to_high;
+  let limits = [0, 50, 100, 500, 1000, 2000];
+
+  let tripPurpose = nameCommute;
+  let scenario = "baseline";
+
+  for (let name of [nameCommute, nameSchool]) {
+    overwritePmtilesSource(
+      $map,
+      name,
+      `${import.meta.env.VITE_RESOURCE_BASE}/layers/v1/${name}.pmtiles`
+    );
+    overwriteLineLayer($map, {
+      id: name,
+      source: name,
+      sourceLayer: name,
+      color: makeColorRamp(["get", scenario], limits, colorScale),
+      width: denseLineWidth,
+      opacity: hoveredToggle(0.5, 1.0),
+    });
+  }
+
+  let show = false;
+
+  function tooltip(feature: MapGeoJSONFeature): string {
+    let baseline = feature.properties.baseline;
+    let gov_target = feature.properties.gov_target;
+    let go_dutch = feature.properties.go_dutch;
+
+    let x = `<h2>Trip purpose: ${
+      tripPurpose == nameCommute ? "commuting" : "school"
+    }</h2>`;
+    x += `<p>Baseline (2011): <b>${baseline.toLocaleString()}</b></p>`;
+    if (baseline == 0) {
+      x += `<p>Government target (2025): <b>${gov_target.toLocaleString()}</b></p>`;
+      x += `<p>Go Dutch: <b>${go_dutch.toLocaleString()}</b></p>`;
+    } else {
+      let multiplier_gov_target = (gov_target / baseline).toFixed(1);
+      let multiplier_go_dutch = (go_dutch / baseline).toFixed(1);
+      x += `<p>Government target (2025): ${gov_target.toLocaleString()} (<b>${multiplier_gov_target}x</b> of baseline)</p>`;
+      x += `<p>Go Dutch: ${go_dutch.toLocaleString()} (<b>${multiplier_go_dutch}x</b> of baseline)</p>`;
+    }
+    return x;
+  }
+
+  function changeStyle() {
+    $map.setPaintProperty(
+      tripPurpose,
+      "line-color",
+      makeColorRamp(["get", scenario], limits, colorScale)
+    );
+  }
+</script>
+
+<Checkbox id={nameCommute} bind:checked={show}>
+  Propensity to Cycle Tool
+  <span slot="right">
+    <HelpButton>
+      <p>
+        The PCT shows roads where cycling levels are predicted to increase,
+        given different assumptions about investments. The layer shows daily
+        counts for commuting to work or school.
```

Comment:
```suggestion
        The PCT shows major links on the transport network on which there is high
        latent demand for cycling under scenarios of change, based on the 'fastest'
        routes from CycleStreets. Counts approximate daily rush hour traffic levels.
```

Reasoning: prediction is not the right word, all depends on the backcasting scenario and deterministic routing. Also captures that these are 'one way' flows which I didn't add because that in itself is confusing and loaded implying direction which the PCT lacks, rush hour traffic is reasonable. The fact that this is the fastest network is important. Do we also want to mention that this is based on old OSM data (not just old OD data)?

Reply:
Thanks! I added a note about old OSM data too and a general caution. Hope we can get the next-gen thing out there soon :)